app/services/offender_service.rb

```diff
@@ -18,6 +18,24 @@ def self.get_offender(offender_no)
     }
   end
 
+  def self.get_community_data(nomis_offender_id)
+    community_info = HmppsApi::CommunityApi::get_offender(nomis_offender_id).deep_symbolize_keys
+    registrations = HmppsApi::CommunityApi::get_offender_registrations(nomis_offender_id).deep_symbolize_keys
+                        .fetch(:registrations)
+                        .select { |r| r.fetch(:active) && r.dig(:registerLevel, :code).starts_with?('M') }
```

Comment:
`registerLevel` is an optional field, and won't exist for every registration (since not every registration is a MAPPA level).

Will this fail if `r.dig(:registerLevel, :code)` is `nil`?

IRB says we'll get:

```
NoMethodError (undefined method `starts_with?' for nil:NilClass)
```